client/src/form/AddNewItemBar.js

```diff
@@ -66,10 +71,10 @@ export default function BasicTextFields(props) {
 
   return (
     <form className={classes.root} noValidate autoComplete="off">
-        <Container align="center" maxWidth="sm" >
-            <Box width="sm" height="sm" className={classes.box} borderRadius={30}>
-                <Container width="90%">
-            <Grid container spacing={1} alignItems="flex-end" align="center" width="90%">
+        <Container align="center" maxWidth="md" >
+            <Box width="sm" height="sm" className={classes.box} borderRadius={50} flexGrow={1}>
+                <Container>
+            <Grid container spacing={1} direction="row" alignItems="flex-end" align="center" className={classes.bar}>
             
                 <Grid item xs={12} md={8}>
                 <TextField 
```

Comment:
can we try this textfield with different variants to see what looks best? I still feel like it would be better if the text was centred vertically and without the line at the bottom

Reply:
Sure centered vertically can do. But I tried with outlined variant it is a square border within circular box...